src/lib/resource/sections/rep-email-address.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = {
+    section: {
+        schema: {
+            $schema: 'http://json-schema.org/draft-07/schema#',
+            type: 'object',
+            additionalProperties: false,
+            properties: {
+                'q-rep-email-address': {
+                    type: 'string',
+                    title: 'Enter your email address',
+                    description:
+                        'We may use this to contact you if we need to clarify something in this application form (optional).',
```

Comment:
@neil-stephen-mcgonigle I have spoken to Graham re the inclusion of the word `form` and he agrees that it's not needed. Can you remove it so sentence reads `We may use this to contact you if we need to clarify something in this application (optional).` I'll update the story with a comment and Graham will update the prototype. Thank you

Reply:
Sorry Julie, 'form' is still there in applicant-enter-your-email-address